IHP/IDE/SchemaDesigner/Parser.hs

```diff
@@ -569,7 +569,16 @@ createFunction = do
     orReplace <- isJust <$> optional (lexeme "OR" >> lexeme "REPLACE")
     lexeme "FUNCTION"
     functionName <- qualifiedIdentifier
-    lexeme "()"
+
+    let functionParameter = do
+        parameterName <- qualifiedIdentifier
+        space
+        parameterType <- qualifiedIdentifier
```

Comment:
It would be more specific to use the `sqlType` parser here. That will accept more complex types as well (e.g. arrays)

```suggestion
        parameterType <- sqlType
```